### PR TITLE
Ft switch to axios 160532247

### PR DESCRIPTION
**Pull request by chepmanu**
#### What does this PR do?
Implements axios for our API calls.

#### Description of tasks to be completed
Our API calls to be handled using axios.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ cd ah-titans-frontend
```
```sh
$ git checkout -b ft-switch-to-axios-160532247
```
```sh
$ npm install
```

#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160532247](https://www.pivotaltracker.com/story/show/160532247)

### CODE REVIEW

**Comment by dcDalin** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Should there be images to represent these components:
  - `src/Login/containers/index.js`
  If these changes are not visual, please update your PR body to include "Skip Visual Diff".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/Login/containers/index.js [hunk 1]
```diff
@@ -1,66 +1,80 @@
...
-      <LoginForm
-        onChange={this.handleChange}
-        onClick={this.handleSubmit}
-        errors={errors}
-        isFetching={isFetching}
-      />
-    );
-  }
+	render() {
+		const { login } = this.props;
```
Comment (jmnyarega) [inline src/Login/containers/index.js:42]: Let's use `componentWillReceiveProps` to handle login props

**Comment by janetnim** [comment]
I have tested your branch and only login works...signup still not working

**Comment by janetnim** [comment]
> I have tested your branch and only login works...signup still not working

**Review by janetnim — APPROVED** [review]

**Review by ronaldndirangu — APPROVED** [review]

**Comment by jmnyarega** [comment]
@janetnim Is the signup action working fine now?

**Comment by janetnim** [comment]
@jmnyarega yes it is working

**Pull request merged**